composer.json

```
@@ -26,7 +26,7 @@
         }
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "^7.3 || ^8.0 || ^8.1",
```

I'm not sure of generic intention, but if we want to stick with 8.1 at most, like some of the packages, then we should have as following
```suggestion
        "php": "^7.3 || ~8.0.0 || ~8.0.0",
```
otherwise the latter should be enough to allow all 8.x
```suggestion
        "php": "^7.3 || ^8.0",
```

---

I agree, thanks for noticing it!